shared/constants.ts

```diff
@@ -1,16 +1,17 @@
-import { Platform } from 'react-native';
-import { VC } from '../types/vc';
+import {Platform} from 'react-native';
+import {VC} from '../types/vc';
 import {
   MIMOTO_HOST,
   ESIGNET_HOST,
   GOOGLE_NEARBY_MESSAGES_API_KEY,
 } from 'react-native-dotenv';
-import { Argon2iConfig } from './commonUtil';
+import {Argon2iConfig} from './commonUtil';
+import {__SessionId} from './GlobalVariables';
```

Comment:
I don't see its usage in the file. There is no change in the file except formatting, please avoid it